.devcontainer/Dockerfile

```diff
@@ -31,8 +31,9 @@ RUN apt-get install --assume-yes \
   "git=1:2.*" \
   "zsh=5.*"
 
-COPY "./scripts/*" "$HOME/"
-RUN curl -fsSL "https://github.com/cashapp/hermit/releases/download/v0.26.1/install.sh" | bash
+COPY ./scripts/ /root/
+
+RUN curl -fsSL "https://github.com/cashapp/hermit/releases/download/stable/install.sh" | bash
```

Comment:
Actually, based on [this comment](https://github.com/cashapp/hermit/issues/345#issuecomment-1326834834), I discovered that Hermit will always install and use the latest from `stable` channel regardless, as defined in `bin/hermit`. So this line can be removed completely, with no side effects.
I'm planning on deleting it in one of my current drafts, if you didn't do it already.

Reply:
No, just cleaning up. Happy to have it removed.